### PR TITLE
[WIP] JS lint fixes

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by nerik** [comment]
Please rebase :)

**Comment by nerik** [comment]
And squash first

**Comment by tiagojsag** [comment]
I will squash once I am done, but for now ignore commit count + messages

File: app/src/components/app.jsx [hunk 1]
```diff
@@ -29,4 +27,12 @@ class App extends Component {
 
 }
 
+App.propTypes = {
+  loading: React.PropTypes.bool,
+  children: React.PropTypes.array,
```
Comment (nerik) [inline app/src/components/app.jsx]: When running this is actually an object, throwing an error. I expect this to happen elsewhere, this code needs to be run first.

**Pull request merged**